crates/common/tedge_config/src/tedge_config_cli/tedge_config_defaults.rs

```diff
@@ -90,6 +94,7 @@ impl From<&TEdgeConfigLocation> for TEdgeConfigDefaults {
         let tmp_path = Path::new(DEFAULT_TMP_PATH);
         let logs_path = Path::new(DEFAULT_LOG_PATH);
         let run_path = Path::new(DEFAULT_RUN_PATH);
+        let data_path = Path::new(DEFAULT_DATA_PATH);
```

Comment:
This is not introduced by this PR, but this pattern is really heavy: a set of `const` used to initialize local values and to finally build a `TEdgeConfigDefault` struct which all fields are prefixed by a  `default` prefix.

Why not simply `Impl Default for TEdgeConfig` ?

Reply:
Makes sense. I'll give that a try.